### PR TITLE
ADR for Metrics collection strategy

### PR DESCRIPTION
**Pull request by bgfoshay**
## What’s changing?

Adding an ADR to discuss a standardized approach for consuming & storing metrics events

## Why?

NAC-922

## This PR has:

- [ ] **Commented source code** (_required on public classes and methods, and elsewhere as appropriate_)

- [x] **Developer documentation** (_for new build/test/API changes or complex portions of the system_)

- [ ] **Automated unit tests** (_to maintain or increase level of code coverage_)

- [ ] **Changes to IAC** (_add any deployment steps that require manual intervention to the draft release notes_)

### CODE REVIEW

File: docs/adr/0034-metrics-events-approach.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# 33. Branching and Release Strategy
```
Comment (dborgen2) [inline docs/adr/0034-metrics-events-approach.md:1]: Nitpick: This should say 34. Metrics Events Approach

File: docs/adr/0033-branch-and-release-strategy.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
+# 33. Branching and Release Strategy
```
Comment (dborgen2) [inline docs/adr/0033-branch-and-release-strategy.md:1]: Question: Is this file supposed to be included in this PR? (I know there's a separate PR for this file)

Comment (bgfoshay) [reply docs/adr/0033-branch-and-release-strategy.md]: No. I've removed it. That should be in a separate branch.

File: docs/adr/0034-metrics-events-approach.md (hunk 1 above)
Comment (bgfoshay) [reply docs/adr/0034-metrics-events-approach.md:1]: Yes, good catch. I'll fix this

**Review by dborgen2 — APPROVED** [review]
👍 LGTM

**Pull request merged**